packages/workbox-precaching/utils/printInstallDetails.mjs

```diff
@@ -7,57 +7,53 @@
 */
 
 import {logger} from 'workbox-core/_private/logger.mjs';
+
 import '../_version.mjs';
 
 /**
  * @param {string} groupTitle
- * @param {Array<PrecacheEntry>} entries
+ * @param {Array<string>} urls
  *
  * @private
  */
-const _nestedGroup = (groupTitle, entries) => {
-  if (entries.length === 0) {
+function _nestedGroup(groupTitle, urls) {
+  if (urls.length === 0) {
     return;
   }
 
   logger.groupCollapsed(groupTitle);
 
-  entries.forEach((entry) => {
-    logger.log(entry._originalInput);
-  });
+  for (const url of urls) {
+    logger.log(url);
+  }
 
   logger.groupEnd();
-};
+}
 
 /**
- * @param {Array<Object>} entriesToPrecache
- * @param {Array<Object>} alreadyPrecachedEntries
+ * @param {Array<string>} urlsToPrecache
+ * @param {Array<string>} urlsAlreadyPrecached
  *
  * @private
- * @memberof module:workbox-precachig
+ * @memberof module:workbox-precaching
  */
-export default (entriesToPrecache, alreadyPrecachedEntries) => {
-  // Goal is to print the message:
-  //    Precaching X files.
-  // Or:
-  //    Precaching X files. Y files were cached and up-to-date.
-
-  const precachedCount = entriesToPrecache.length;
-  const alreadyPrecachedCount = alreadyPrecachedEntries.length;
-  let printText =
-    `Precaching ${precachedCount} file${precachedCount === 1 ? '' : 's'}.`;
-  if (alreadyPrecachedCount > 0) {
-    printText += ` ${alreadyPrecachedCount} ` +
-      `file${alreadyPrecachedCount === 1 ? ' is' : 's are'} already cached.`;
-  }
+export default (urlsToPrecache, urlsAlreadyPrecached) => {
```

Comment:
Same comment as above, can we make this a named export?

Reply:
Changed.